include/mbgl/map/transform.hpp

```diff
@@ -52,6 +53,10 @@ class Transform : private util::noncopyable {
     void startRotating();
     void stopRotating();
 
+    // Projection
+    void pixelForLatLng(const LatLng latLng, double &x, double &y) const;
```

Comment:
Reason I did this was because calling code (e.g. iOS) would have to know about `vec2` type in this case.